### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this project I have: 
- Opened the project Bookstore CMS
- Used the information detailed in Zeplin to style the website

### CODE REVIEW

**Review by eliasecasta — CHANGES_REQUESTED** [review]
## Status:  Invalid 🔴 

Hi, @emmanuellekamwa, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors, please fix them before requesting another review, have a nice day, and good luck!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,19 +1,70 @@
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const FETCH_BOOK_SUCCESS = 'bookStore/books/FETCH_BOOK_SUCCESS';
 
 const initialState = [];
```
Comment (Davidosky007) [inline src/redux/books/books.js:5]: * I get an error in the console when i run `npm start` kindly fix this error.  HINT 💭: 
*  If you look at the console error message you will see that you are trying to import `redux/books/books.js` as `redux/books/Books.js` the `Books.js` in uppercase is the problem 😢 you can correct this by using the right file name 👍 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @emmanuellekamwa 👋, 

Good job so far! 💪 
* Descriptive commit messages ✔️ 
* Proper git-flow pattern ✔️ 
* No linter errors ✔️ 
* Well written PR ✔️ 
* Professional readme file ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @emmanuellekamwa, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**